app/models/agents/website_agent.rb

```diff
@@ -166,6 +169,27 @@ def validate_options
       validate_web_request_options!
     end
 
+    def validate_consider_http_success_option!
+      consider_success = options["http_success_codes"]
+      if consider_success != nil
```

Comment:
I still recommend `consider_success.present?` because users may clear the field in the UI, which is an empty string, not nil.  `present?` returns false on an empty string as well as false and nil.